src/Exceptions/HttpURITooLong.php

```diff
@@ -8,4 +8,3 @@ class HttpURITooLong extends HttpException {
     public const DESCRIPTION = "Indicates that the server is refusing to service the request because the request-target is longer than the server is willing to interpret.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/414";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces